packages/react-icons/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "spnels-fluent-system-test0000",
```

Comment:
@fluentui/react-icons

Reply:
We should probably get an owner from Fluent UI React to ensure that the namespace works with their current expectations/requirements (if we haven't already).

Reply:
Yeah, `@fluentui/react-icons` is the expected namespace for this package